### ISSUE TITLE
Features requests: the discovery of new channels and bookmark placement.

### ISSUE THREAD

**Issue by Esnaiper** [crackededed/Xtra#81]
First at all, i just discovered xtra and im loving it xqcL dude.

-I miss a lot the "Sort by Viewers (low to high)" in categories to discover new channels. 
-Would be cool a way to use the bookmart directly in the player.
-I miss the "recommended to you" and "channels you would like" a little bit too but i think that would be hard to put it.
-This thing it would be hard to put it too, its the option to see which channel an emote its from.

In conclusion: i just want the "Sort by Viewers (low to high)" Sadge. Btw there is a way to financially or practically support the project?
